snakemake_executor_plugin_slurm/__init__.py

```
@@ -92,6 +128,36 @@
         self._fallback_partition = None
         self._preemption_warning = False  # no preemption warning has been issued
 
+        def clean_old_logs(logdir, age_cutoff):
```

Ruff

Redefinition of unused `clean_old_logs` from line 30

(F811)